editor.planx.uk/src/ui/editor/Dashboard.tsx

```diff
@@ -0,0 +1,21 @@
+import Box from "@mui/material/Box";
+import { styled } from "@mui/material/styles";
+import { HEADER_HEIGHT } from "components/Header";
+import React, { PropsWithChildren } from "react";
+
+const Root = styled(Box)(({ theme }) => ({
+  backgroundColor: theme.palette.background.default,
+  color: theme.palette.text.primary,
+  display: "flex",
+  flexDirection: "row",
+  width: "100%",
+  minHeight: `calc(100vh - ${HEADER_HEIGHT}px)`,
+  "& > .MuiContainer-root": {
```

Comment:
I think we should be able to import this classname from MUI 👍

Reply:
Good spot, applied